docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -225,35 +251,44 @@ client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client c
 root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs
 ```
 </TabItem>
-  <TabItem label="Connect to the Proxy Service">
+<TabItem label="Connect to the Proxy Service" options="Executable">
 
 Set `addr` to  your Proxy Service address with port `443`.
 
 Set `identity` to the identity file generated earlier:
 
 ```toml
 [teleport]
-addr = "mytenant.teleport.sh:443"
+addr = "teleport.example.com:443"
 identity = "/var/lib/teleport/plugins/slack/auth.pem"
 ```
-  </TabItem>
+</TabItem>
+<TabItem label="Connect to the Proxy or Auth Service" scope="cloud" options="Helm Chart">
+
+**`address`**: Include the hostname and port of your Teleport Proxy or Auth Service.
+(e.g., `teleport.example.com:443`).
+
+**`identitySecretName`**: Fill in the `identitySecretName` field with the name
+of the Kubernetes secret you created earlier.
```

Comment:
I'm not sure what this refers to—there are no instructions earlier in the page for creating a Kubernetes secret.

Reply:
It's in `docs/pages/includes/plugins/identity-export.mdx` under the Helm Chart dropdown